pkgs/build-support/build-mozilla-mach/default.nix

```diff
@@ -318,12 +318,15 @@ buildStdenv.mkDerivation {
     ++
       lib.optionals
         (
-          lib.versionAtLeast version "141.0.2"
+          (lib.versionAtLeast version "141.0.2" && lib.versionOlder version "145.0")
           || (lib.versionAtLeast version "140.2.0" && lib.versionOlder version "141.0")
         )
         [
           ./142-relax-apple-sdk.patch
         ]
```

Comment:
This whole thing is gone on staging-next. Should have gotten backported.

f2895523473dab52d67d77779511177d288fab61

Except master has 15.5 and release-25.05 has 15.2, so maybe not.